.github/workflows/celerity_ci.yml

```diff
@@ -28,65 +28,84 @@ jobs:
       fail-fast: false
       matrix:
         include:
```

Comment:
I think we should add a note here that whenever this is updated, `docs/platform-support.md` should also be updated!

Reply:
Done.